contrib/docker/apache/conf-available/remoteip.conf

```diff
@@ -0,0 +1,4 @@
+RemoteIPHeader X-Real-IP
+RemoteIPTrustedProxy 10.0.0.0/8
```

Comment:
TODO: want to include all of [RFC1918](https://en.wikipedia.org/wiki/Private_network#Private_IPv4_addresses) here

Reply:
done!